app/Models/Ldap.php

```diff
@@ -286,7 +286,7 @@ static function findLdapUsers($base_dn = null)
         // Clean up after search
         $result_set['count'] = $global_count;
         $results = $result_set;
-        ldap_control_paged_result($ldapconn, 0);
+        @ldap_control_paged_result($ldapconn, 0);
```

Comment:
Didn't this already get merged in yesterday or the day before?

Reply:
D'oh! Yup, it did. I'll rebase.